### PR TITLE
Adjust custom css for support type drawer

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change
Recent changes to support type on confluence caused some unexpected display issues in Confluence. This PR is a quick fix for that

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-0

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Before merge to main

- [ ] OHS demo complete
- [ ] Ready to create production PR

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: frontend/src/components/SupportTypeDrawer.scss [hunk 1]
```diff
@@ -1,7 +1,26 @@
...
+.ttahub-drawer--objective-support-type-guidance .rdw-editor-wrapper > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(64) > div:nth-child(1) > span:nth-child(3) > span:nth-child(1),
+.ttahub-drawer--objective-support-type-guidance .rdw-editor-wrapper > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(3),
+.ttahub-drawer--objective-support-type-guidance .rdw-editor-wrapper > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(4),
+.ttahub-drawer--objective-support-type-guidance .rdw-editor-wrapper > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(1),
+.ttahub-drawer--objective-support-type-guidance .rdw-editor-wrapper > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(64) > div:nth-child(1) {
```
Comment (thewatermethod) [inline frontend/src/components/SupportTypeDrawer.scss:23]: absolutely disgusting

Comment (AdamAdHocTeam) [reply frontend/src/components/SupportTypeDrawer.scss:23]: none of this lol

**Review by AdamAdHocTeam — APPROVED** [review]
I looked at a few drawers they seem to be displaying correctly. Not sure if there was anything specific to test.

**Comment by thewatermethod** [comment]
I'm gonna hold off merging this and go with the updated RSS parsing (#2366 ) instead of hotfixing our old implementation, but I'll leave this open just for now

**Pull request closed without merging**